pkg/controller/windowsmachine/windowsmachine_controller.go

```diff
@@ -303,13 +309,15 @@ func (r *ReconcileWindowsMachine) Reconcile(request reconcile.Request) (reconcil
 					r.recorder.Eventf(machine, core.EventTypeWarning, "MachineDeletionFailed", "Machine %v deletion failed: unable to delete Machine object: %v", machine.Name, err)
 					return reconcile.Result{}, err
 				}
+				log.Info("machine has been remediated by deletion", "name", machine.GetName())
 				r.recorder.Eventf(machine, core.EventTypeNormal, "MachineDeleted", "Machine %v has been remediated by requesting to delete Machine object", machine.Name)
 				return reconcile.Result{}, nil
 			}
 			// version annotation exists with a valid value, node is fully configured, do nothing.
```

Comment:
Should we add a log message/ event here ?

Reply:
Sure I can add a log message here. Do you want it to be `log.Info()` or `log.V(1).Info()`? I don't think an event is warranted unless you can think of a reason a customer would want to see it.